package.json

```diff
@@ -52,7 +54,7 @@
   },
   "lint-staged": {
     "*.{js,ts}": [
-      "npm run typecheck",
+      "npm run typecheck -- --lib ESNext",
```

Comment:
I found lint-staged was ignoring the ts config when invoked by husky, turns out other people have seen that too https://github.com/okonet/lint-staged/issues/825. Would be nice to find a better solve than passing the opts here but this unblocked me for now. 

NB without passing the lib here you get an error pre-push stating that lib must be ES2021 for higher to use `String.replaceAll`

Reply:
#11 Created to track & fix this properly.